register.php

```diff
@@ -0,0 +1,92 @@
+<?php
+  require("model/DbConnection.php");
+  require("model/features.php");
+  // If already loggedin redirect to submit page.
+  if (isset($_SESSION['loggedIn']) && $_SESSION['loggedIn'] == TRUE)
+    header("Location:index.php");
+
+  // Connect with Credential table database.
+  $db = new DbConnection();
+  $feature= new features();
+
+  // Checks first if submit is clicked
+  if (isset($_POST["name"]) && isset($_POST["mailId"]) && isset($_POST["userId"]) && isset($_POST["pass"])) {
+    // If id and pass fields are not empty.
+    if ($_POST['name'] != "" && $_POST["mailId"] != "" && $_POST['userId'] != "" && $_POST["pass"] != "") {
+      if (!$feature->onlyAlpha($_POST['name'])){?>
+        <h4 class='error-div'>Invalid name</h4>
+      <?php
+      }
+      if (!$feature->validMailId($_POST['mailId'])){?>
+        <h4 class='error-div'>Invalid email address.</h4>
+      <?php
+      }
+      if (!$feature->validUserId($_POST['userId'])){?>
+        <h4 class='error-div'>User Id should only contain alphabet numbers and space</h4>
+      <?php
+      }
+      // If user is not available in db
+      if ($db->existsUserId($_POST['userId'])){?>
+        <h4 class='error-div'>User Id is unavailable</h4>
+      <?php
+      }
+      // If mail id is not available in db
+      if ($db->existsMailId($_POST['mailId'])){?>
+        <h4 class='error-div'>Mail Id is unavailable</h4>
+      <?php
+      }
+      // If all fileds are valid then insert data into database
+      else {
+        try {
+          date_default_timezone_set("Asia/Kolkata");
+          $sql = "INSERT INTO `oauth-users` (`oauth_provider`, first_name, last_name, email, user_id, pass,modified) values('postit','" . explode(' ', $_POST['name'], 2)[0] . "','" . explode(' ', $_POST['name'] . " ", 2)[1] . "','" . $_POST["mailId"] . "','" . $_POST['userId'] . "','" . $_POST['pass'] . "','" . date("y-m-d h:i:s") . "')";
+          $db->conn->query($sql);?>
```

Comment:
Put this in model. Follow mvc structure. All database query must be in model file.